reaktive/src/commonMain/kotlin/com/badoo/reaktive/utils/ObjectReference.kt

```diff
@@ -0,0 +1,6 @@
+package com.badoo.reaktive.utils
+
+internal expect class ObjectReference<T>(initialValue: T) {
```

Comment:
Maybe make it public? Can be useful to write own operators. Just add some kdoc and alert like "if you are not sure, use AtomicReference".